src/tools/validator/custom-validations/validate-restriction-ids.ts

```diff
@@ -18,15 +19,23 @@ export const validRestrictionIds = (
     ];
   }
 
-  const restrictionData = JSON.parse(
-    fs.readFileSync(sourcePath, { encoding: 'utf8' })
-  );
+  let restrictionData: Restrictions | undefined;
+  try {
+    restrictionData = JSON.parse(
+      fs.readFileSync(sourcePath, { encoding: 'utf8' })
+    );
+    /* eslint-disable-next-line */
+  } catch (e) {}
+
+  if (restrictionData === undefined) {
+    return [`Restrictions file '${sourcePath}' could not be parsed to JSON.`];
+  }
 
-  const restrictionIds = (restrictionData.values as any[]).map(
+  const restrictionIds = restrictionData.values.map(
     (value: any) => value.identifier
```

Comment:
```suggestion
    (value) => value.identifier
```
With functions like map I often use a shorthand x. It often makes the formatter place things on the same line

```suggestion
    (x) => x.identifier
```